docs/managed-datahub/approval-workflows.md

```diff
@@ -0,0 +1,184 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# Approval Workflows
+
```

Comment:
We need to make it clear that this functionality is currently incubating, only available for datasets.